junixsocket-common/src/main/java/org/newsclub/net/unix/NativeUnixSocket.java

```diff
@@ -20,25 +20,61 @@
 import java.io.FileDescriptor;
 import java.io.IOException;
 import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
 import java.net.InetSocketAddress;
-
+import java.net.Socket;
+import java.net.ServerSocket;
 /**
  * JNI connector to native JNI C code.
  * 
  * @author Christian Kohlschütter
  */
 final class NativeUnixSocket {
+  private static native void registerNatives();
   private static boolean loaded = false;
+  private static Method getMethod(Class<?> klass, String name, Class<?>...parameterTypes) {
```

Comment:
I'm not sure if junixsocket has a specific language policy.  If so, it may be that the English "class" is preferred over the German "klass".  Not that it matters much compile wise. ;)